src/core/optimization_container.jl

```diff
@@ -409,6 +416,28 @@ function get_aux_variable_keys(container::OptimizationContainer)
     return collect(keys(container.aux_variables))
 end
 
+##################################### DualVariable Container ################################
+function add_dual_container!(
```

Comment:
Wherever this is called we need to check if the container is compatible i.e., check if its MILP